### PR TITLE
fix view code buttons

### PR DESCRIPTION
**Pull request by susanev**
## overview
fixing up the view code buttons so they don't look so tiny next to the large deploy with pulumi button
fixes https://github.com/pulumi/registry/issues/1822

### before
Screen Shot 2023-01-03 at 5 43 53 PM

### after
Screen Shot 2023-01-03 at 6 09 59 PM

### CODE REVIEW

File: tools/mktutorial/templates/tutorial.tmpl [hunk 1]
```diff
@@ -19,7 +19,7 @@ layout: how-to-guide
 <!-- To change it, please see https://github.com/pulumi/docs/tree/master/tools/mktutorial. -->
 
 <p class="mb-4 flex">
-    <a class="flex flex-wrap items-center rounded text-xs text-white bg-blue-600 border-2 border-blue-600 px-2 mr-2 whitespace-no-wrap hover:text-white" style="height: 32px" href="{{GitHubURL}}" target="_blank">
+    <a class="flex flex-wrap items-center rounded-md text-lg text-white bg-blue-600 border-2 border-blue-600 px-2 mr-2 whitespace-no-wrap hover:text-white" style="height: 45px; font-family: 'Gilroy'; " href="{{GitHubURL}}" target="_blank">
```
Comment (susanev) [inline tools/mktutorial/templates/tutorial.tmpl:22]: this is the thing to review, everything else is generated

Comment (cnunciato) [reply tools/mktutorial/templates/tutorial.tmpl:22]: I think for this, you can use the `font-display` class as it’s defined as Gilroy in the Tailwind config.

Comment (susanev) [reply tools/mktutorial/templates/tutorial.tmpl:22]: ah cool, didnt know

**Review by cnunciato — APPROVED** [review]
Thanks! 🎉

**Review by kimberleyamackenzie — APPROVED** [review]

**Pull request merged**